### DISCUSSION TITLE
zh/modes/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#8559] [Docs]
# zh/modes/

从培训到跟踪，Ultralytics ，充分利用YOLOv8 。获取每种支持模式的见解和示例，包括验证、导出和基准。

https://docs.ultralytics.com/zh/modes/

**Comment by Monkey1GIt** [comment]
The Chinese web page translation is very bad, for example, "train" should be "训练", but translated into "火车"

And can not switch to English page, click the EN button, will still switch to Chinese page

**Reply by glenn-jocher** [reply]
Hey there! 😊 Thanks for bringing this to our attention. We're sorry for the inconvenience caused by the translation errors and the issue with the language switch button. We're constantly working on improving our documentation and its accessibility, including accurate translations.

For the translation issue, we'll look into it and aim to correct those errors, like "train" being incorrectly translated. Your feedback is invaluable in helping us enhance the user experience.

Regarding the language switch issue, we're investigating this and will work on a fix. In the meantime, if you're still unable to switch to the English page, you might try clearing your browser's cache or using a different browser as a temporary workaround.

If you have any more feedback or need further assistance, please don't hesitate to reach out. Thanks for your support and understanding! 🙏
